Extension/i18n/esn/package.i18n.json

```diff
@@ -171,7 +171,7 @@
 	"c_cpp.configuration.addNodeAddonIncludePaths.description": "Agregue rutas de acceso de inclusión de nan y node-addon-api cuando sean dependencias.",
 	"c_cpp.configuration.renameRequiresIdentifier.description": "Si es true, \"Cambiar el nombre del símbolo\" requerirá un identificador de C/C++ válido.",
 	"c_cpp.configuration.autocompleteAddParentheses.description": "Si es true, la opción de autocompletar agregará \"(\" de forma automática después de las llamadas a funciones, en cuyo caso puede que también se agregue \")\", en función del valor de la configuración de \"editor.autoClosingBrackets\".",
-	"c_cpp.configuration.filesExclude.description": "Configure patrones globales para excluir carpetas (y archivos si se cambia \"C_Cpp.exclusionPolicy\"). Estos son específicos de la extensión de C/C++ y se agregan a \"files.exclude\", pero a diferencia de \"files.exclude\" no se quitan de la vista del explorador. Más información acerca de los patrones de globales [aquí](https://code.visualstudio.com/docs/editor/codebasics#_advanced-search-options).",
+	"c_cpp.configuration.filesExclude.description": "Configure patrones globales para excluir carpetas (y archivos si se cambia \"C_Cpp. exclusionPolicy\"). Estos son específicos de la extensión de C/C++ y se agregan a \"files. Exclude\", pero a diferencia de \"files. Exclude\" no se quitan de la vista del explorador. Más información acerca de los patrones de globales [aquí](https://code.visualstudio.com/docs/editor/codebasics#_advanced-search-options).",
```

Comment:
This is wrong.  Ultimately, we need a fix for: https://github.com/microsoft/vscode/issues/125774

Reply:
Yes, I pinged the loc team yesterday on that.